tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java

```diff
@@ -149,7 +149,18 @@ public static void main(String[] args) throws IOException {
       }
       Webfiles manifest = loadWebfilesPbtxt(Paths.get(args[i]));
       for (WebfilesSource src : manifest.getSrcList()) {
-        webfiles.put(Webpath.get(src.getWebpath()), Paths.get(src.getPath()));
+        String webpath = src.getWebpath();
+        Path srcPath = Paths.get(src.getPath());
+
+        webfiles.put(Webpath.get(webpath), srcPath);
+
+        if (webpath.startsWith("/polymer/externs")) {
```

Comment:
Check `"/polymer/externs/"` (with trailing slash) to prevent spliced-in
paths like `"/polymer/externs-foo"`?